Dockerfile

```diff
@@ -121,6 +121,8 @@ FROM base AS pythonbuild
 ARG CONAN_VERSION
 ARG CONAN2_VERSION
 ARG PIP_VERSION
+# PYENV_GIT_TAG is consumed as described here:
+# https://github.com/pyenv/pyenv-installer/blob/63a9e6a216796aeba2535a3bac8e79ba5d95166d/README.rst?plain=1#L22.
```

Comment:
Hmm, I wonder whether this actually ever worked, as we're not setting `PYENV_GIT_TAG` as an `ENV` here, but as an `ARG` (which is never used)...

Reply:
Yeah, let's go with your less invasive solution for now.

Reply:
So, as the other PRs turned out not needed, shall we proceed with this one?